modules/user/client-react/components/ActivateUser.native.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ActivityIndicator, StyleSheet, Text, View } from 'react-native';
+import { translate } from '@module/i18n-client-react';
+
+class ActivateUser extends React.PureComponent {
+  static propTypes = {
+    t: PropTypes.func
+  };
+
+  render() {
+    const { t } = this.props;
+
+    return (
+      <View style={styles.container}>
+        <ActivityIndicator size="large" color="#0000ff" />
+        <Text style={styles.text}>
+          {t('activateUser')}
+          ...
```

Comment:
Why are these dots here, and not in a file with localization?

Reply:
Done